demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -181,7 +181,8 @@ def validate_pack_meta_file(self):
             self._is_approved_tags(),
             self._is_price_changed(),
             self._is_approved_tags(),
-            self._is_valid_support_type()
+            self._is_valid_support_type(),
+            self.is_right_usage_of_usecase_tag()
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
            self.is_right_usage_of_usecase_tag(),
```